wp-modules/pattern-data-handlers/pattern-data-handlers.php

```diff
@@ -356,7 +356,7 @@ function construct_pattern_php_file_contents( $pattern_data ) {
 	$file_contents = '<?php
 /**
  * Title: ' . addcslashes( $pattern['title'], '\'' ) . '
- * Slug: ' . $pattern['name'] . '
+ * Slug: ' . $pattern['slug'] . '
```

Comment:
This meant that any edit, even to content, changes the slug to be the same as the name.

Reply:
Great find

Reply:
Thanks, Phil!